net/net/inc/TMessage.h

```diff
@@ -39,15 +39,15 @@ friend class TPSocket;
 friend class TXSocket;
 
 private:
-   TList   *fInfos;       //List of TStreamerInfo used in WriteObject
-   TBits    fBitsPIDs;    //Array of bits to mark the TProcessIDs uids written to the message
-   UInt_t   fWhat;        //Message type
-   TClass  *fClass;       //If message is kMESS_OBJECT pointer to object's class
-   Int_t    fCompress;    //Compression level and algorithm
-   char    *fBufComp;     //Compressed buffer
-   char    *fBufCompCur;  //Current position in compressed buffer
-   char    *fCompPos;     //Position of fBufCur when message was compressed
-   Bool_t   fEvolution;   //True if support for schema evolution required
+   TList *fInfos{nullptr};     // List of TStreamerInfo used in WriteObject
```

Comment:
Minor.  Can you keep the alignment of the data member name?  Thanks.

Reply:
It is that clang-format doing automatically. But I can revert changes.

Reply:
clang-format is result 'there' is sub-optimal :)